rest-server/README.md

```diff
@@ -463,3 +463,86 @@ Configure the rest server ip and port in [service-deployment/clusterconfig.yaml]
       "error": "JobExecuteError",
       "message": "job execute error"
     }
+
+11. `GET virtual-clusters/`
+
+    Get the list of vcs.
+
+    *Request*
+    ```
+    GET /api/v1/virtual-clusters
+    ```
+
+    *Response if succeeded*
+    ```
+    {
+      "vc1": 
+      {
+      }
+      ...
+    }
+    ```
+
+    *Response if a server error occured*
+    ```
+    Status: 500
+
+    {
+      "error": "GetVcListError",
+      "message": "get vc list error"
+    }
+    ```
+    
+12. `GET virtual-clusters/:vcName`
+
+    Get vc status in the system.
+
+    *Request*
+    ```
+    GET /api/v1/virtual-clusters/:vcName
+    ```
+
+    *Response if succeeded*
+    ```
+    {
+      "capacity":0,
```

Comment:
Any field that comes from the user's config?

Reply:
Please define the input/output of the API, and then have some rounds of discussion before the actual implementation.